src/plugins/intel_gpu/src/graph/impls/ocl/kernel_selector_helper.h

```diff
@@ -253,6 +253,23 @@ inline bool broadcastable(const ov::PartialShape& first_pshape, const ov::Partia
     return true;
 }
 
+inline bool broadcastableFromTo(const ov::PartialShape& from, const ov::PartialShape& to) {
+    if (from.is_dynamic() || to.is_dynamic())
+        return false;
+
+    if (from.size() > to.size())
+        return false;
+
+    size_t min_size = from.size();
+
+    for (size_t i = 0; i < min_size; ++i) {
+        if (!(from[i] == 1 || from[i] == to[i])) {
+            return false;
+        }
+    }
+    return true;
+}
```

Comment:
Why do we need this new "braodcastableFromTo" not the original one above? (Line 239)

Reply:
original one check bi-directional, but for this issue, one-direction check is needed.